pkg/network/mptpool/pool.go

```diff
@@ -14,13 +16,19 @@ type Pool struct {
 	lock   sync.RWMutex
 	hashes map[util.Uint256][][]byte
 
-	// TODO: retransmission
+	resendThreshold time.Duration
+	resendFunc      func(map[util.Uint256]bool)
+
+	batchCh          chan struct{}
+	retransmissionOn *atomic.Bool
```

Comment:
Do we really need retransmissions when every Peer already has its own ticking thing in `StartProtocol`? And it just can request MPT nodes instead of blocks?

Reply:
Fixed.